examples/chip-tool/commands/discover/DiscoverCommissionablesCommand.h

```diff
@@ -25,13 +25,18 @@ class DiscoverCommissionablesCommandBase : public CHIPCommand, public chip::Cont
 public:
     DiscoverCommissionablesCommandBase(const char * name, CredentialIssuerCommands * credsIssuerConfig) :
         CHIPCommand(name, credsIssuerConfig)
-    {}
+    {
+        AddArgument("discover-once", 0, 1, &mDiscoverOnce);
```

Comment:
Might be worth adding a docstring that explains what this does.

Reply:
Maybe even some documentation update in https://project-chip.github.io/connectedhomeip-doc/guides/chip_tool_guide.html#discover-actions